src/components/CheckingAccountsList.jsx

```diff
@@ -0,0 +1,35 @@
+
+import {useState,useEffect ,useContext} from 'react';
+import { getAllCheckingAccounts } from '../api';
+import { myContext } from '../context/Authcontext';
+import { AccountCard } from './AccountCard';
+
+
+export const CheckingAccountsList = () => {
+
+    
+    const {user} = useContext(myContext); 
+    const [checkingAccounts,setCheckingAccounts] = useState([]);
+    const [loading,setLoading] = useState(true); 
+
+    useEffect(()=>{
+
+      getAllCheckingAccounts(user).then(response => {
```

Comment:
I'd set up the Axios instance to avoid passing the corresponding user every time you need to call one of the API functions.

Reply:
Hint: You can use the data of the logged-in user to set it up.

Reply:
There may be a problem with that, because apparently I can´t use the `useContext()` to read the token value outside a react component, so if I do that there will be an error

Reply:
Did you try storing the token in the `sessionStorage`?

Reply:
Done!